fix(worker): Backport fix for #58 to banch 7.0

agb80 opened the pull request.

Commit:
Migration to be odoo master compliant

agb80 committed:
fix(worker): Backport fix for #58 to banch 7.0

agb80 committed:
fix(mapper): Mapp zero values as numeric instead of False

Zero is not Null neither False. A new validation was needed to ensure not send a false False mapped

value.

guewen commented on connector/unit/mapper.py:
```diff
@@ -142,7 +142,7 @@ def convert(field, conv_type):
     def modifier(self, record, to_attr):
         value = record[field]
-        if not value:
+        if not value and value != 0:
```
there's the same issue with `value == ''` so maybe we should totally change this condition

agb80 replied:
What about try with something like this?
`if value is not None:`

Commit:
fix(migrations): delete uncessary migration

this migration is for 2.2.0 version but we are in 2.3.0 and we have been working without this
migration time ago, and besides that this migrations fails when it runs
